triage/Dockerfile

```diff
@@ -22,7 +22,14 @@ COPY . /test-infra/
 
 # Build Triage
 RUN cd /test-infra/triage/ && \
-    env GOOS=linux go build -v -o triage ./main.go
+    env GOOS=linux \
+        GOARCH=amd64 \
```

Comment:
this is unnecessary, we should be able to just match the host arch

Reply:
Removed